pulsar-client/pom.xml

```diff
@@ -74,6 +74,17 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
This should be added to the client shading as well. Also in all the other shading config they need to reuse the same list of the client shading conf.

Reply:
oops.. I missed it. I will add it.